src/pkg/packaging/deb/dotnet-sharedframework-debian9_config.json

```diff
@@ -0,0 +1,41 @@
+{
+    "maintainer_name":"Microsoft",
+    "maintainer_email": "dotnetcore@microsoft.com",
+
+    "package_name": "%SHARED_FRAMEWORK_DEBIAN_PACKAGE_NAME%",
+    "install_root": "/usr/share/dotnet",
+
+    "short_description": "%SHARED_FRAMEWORK_BRAND_NAME% %SHARED_FRAMEWORK_NUGET_NAME% %SHARED_FRAMEWORK_NUGET_VERSION%",
+    "long_description": ".NET Core is a development platform that you can use to build command-line applications, microservices and modern websites. It is open source, cross-platform and is supported by Microsoft. We hope you enjoy using it! If you do, please consider joining the active community of developers that are contributing to the project on GitHub (https://github.com/dotnet/core). We happily accept issues and PRs.",
+    "homepage": "https://dotnet.github.io",
+
+    "release":{
+        "package_version":"1.0.0.0",
```

Comment:
This is the default value that always gets set to the correct product version during the package build step. see https://github.com/dotnet/core-setup/blob/master/src/pkg/packaging/deb/package.targets#L178 .